### PR TITLE
feat: Develop TvlChart component

### PR DESCRIPTION
**Pull request by juntaepark**

### CODE REVIEW

File: packages/web/src/components/common/select-tab/SelectTab.tsx [hunk 1]
```diff
@@ -1,24 +1,45 @@
...
-          key={idx}
-          className={cx({ selected: type === selectType })}
-          onClick={() => onClick(type)}
-          disabled={type === selectType}
-        >
-          {type}
-        </SelectButton>
-      ))}
+      {list.map((type, idx) =>
+        className === "chart-tab" ? (
```
Comment (jinoosss) [inline packages/web/src/components/common/select-tab/SelectTab.tsx:23]: I don't think it's explicit to handle rendering conditions by class name.
What about using explicit `type` or `enum`?

File: packages/web/src/components/common/select-tab/SelectTab.styles.ts [hunk 2]
```diff
@@ -23,3 +23,18 @@ export const SelectButton = styled.button`
...
   }
 `;
+
+export const ChartSelectButton = styled.button`
+  ${mixins.flexbox("row", "center", "center")};
+  ${fonts.body10};
+  width: 60px;
+  height: 37px;
+  padding: 0px 8px;
+  border-radius: 4px, 0px, 0px, 4px;
```
Comment (jinoosss) [inline packages/web/src/components/common/select-tab/SelectTab.styles.ts:33]: border-radius doesn't seem to be applied.
`border-radius: 4px 0px 0px 4px;` seems to be correct.
Also, it looks like you intended for only the border-radius of the left part to be applied, I'll have to check if the same border-radius should be applied for all items.
(My guess is that the radius of the first item would be applied to the left, and the last item to the right).

File: packages/web/src/components/dashboard/tvl-chart-info/TvlChartInfo.styles.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import mixins from "@styles/mixins";
+import { fonts } from "@constants/font.constant";
+
+export const TvlChartInfoWrapper = styled.div`
+  ${mixins.flexbox("column", "flex-start", "flex-start")};
+  background-color: ${({ theme }) => theme.color.background01};
+  border-radius: 8px;
+  padding: 24px 24px 23px 24px;
+  gap: 16px;
+  width: 620px;
```
Comment (jinoosss) [inline packages/web/src/components/dashboard/tvl-chart-info/TvlChartInfo.styles.ts:11]: The chart area doesn't seem to be flexible.

File: packages/web/src/components/dashboard/volume-chart-info/VolumeChartInfo.styles.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+import mixins from "@styles/mixins";
+import { fonts } from "@constants/font.constant";
+
+export const VolumeChartInfoWrapper = styled.div`
+  ${mixins.flexbox("column", "flex-start", "flex-start")};
+  background-color: ${({ theme }) => theme.color.background01};
+  border-radius: 8px;
+  padding: 24px 24px 23px 24px;
+  gap: 16px;
+  width: 620px;
```
Comment (jinoosss) [inline packages/web/src/components/dashboard/volume-chart-info/VolumeChartInfo.styles.ts:11]: The chart area doesn't seem to be flexible.

File: packages/web/src/containers/tvl-chart-container/TvlChartContainer.tsx [hunk 5]
```diff
@@ -1,7 +1,67 @@
...
+    TVL_CHART_TYPE["7D"],
+  );
+
+  const { data: tvlPriceInfo } = useQuery<TvlPriceInfo, Error>({
+    queryKey: ["tvlPriceInfo"],
+    queryFn: () => fetchTvlPriceInfo(),
+    initialData: initialTvlChartPriceInfo,
+  });
+
+  const changeTvlChartType = useCallback((newType: string) => {
```
Comment (jinoosss) [inline packages/web/src/containers/tvl-chart-container/TvlChartContainer.tsx:39]: How do you think this code?
```
const tvlChartType = Object.values(TVL_CHART_TYPE).find(type => type === newType) || TVL_CHART_TYPE["7D"];
setTvlChartType(tvlChartType);
```

**Review by jinoosss — COMMENTED** [review]
I left a few comments.

I'd like you to add a file for jest testing and a file for storybook testing.

File: packages/web/src/components/dashboard/volume-chart-info/VolumeChartInfo.styles.ts (hunk 4 above)
Comment (juntaepark) [reply packages/web/src/components/dashboard/volume-chart-info/VolumeChartInfo.styles.ts:11]: I'll add. thank you.

File: packages/web/src/components/dashboard/tvl-chart-info/TvlChartInfo.styles.ts (hunk 3 above)
Comment (juntaepark) [reply packages/web/src/components/dashboard/tvl-chart-info/TvlChartInfo.styles.ts:11]: I'll add. thank you.

File: packages/web/src/containers/tvl-chart-container/TvlChartContainer.tsx (hunk 5 above)
Comment (juntaepark) [reply packages/web/src/containers/tvl-chart-container/TvlChartContainer.tsx:39]: I think the code is simpler and nicer!

File: packages/web/src/components/common/select-tab/SelectTab.styles.ts (hunk 2 above)
Comment (juntaepark) [reply packages/web/src/components/common/select-tab/SelectTab.styles.ts:33]: thank you Let's check again.

File: packages/web/src/components/common/select-tab/SelectTab.tsx (hunk 1 above)
Comment (juntaepark) [reply packages/web/src/components/common/select-tab/SelectTab.tsx:23]: Rendering by type or enum would be nice.

**Comment by juntaepark** [comment]
I edited the part you commented on, please check it out.

**Review by jinoosss — APPROVED** [review]
LGTM !

**Pull request merged**